test/e2e/framework/framework.go

```diff
@@ -261,10 +261,10 @@ func (f *Framework) BeforeEach() {
 		var err error
 		var nodeMode NodesSet
 		switch TestContext.GatherKubeSystemResourceUsageData {
-		case "master":
-			nodeMode = MasterNodes
-		case "masteranddns":
-			nodeMode = MasterAndDNSNodes
+		case "controlplane":
```

Comment:
Isn't this a functional and/or API change?

Whoever sets TestContext.GatherKubeSystemResourceUsageData must change their invocation. This needs to be announced in the changelog.

I prefer to accept "master" as an alias for "controlplane" until everyone had a chance to adapt.